setup-bb-worker.sh

```diff
@@ -3,17 +3,21 @@
 set -e
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+if [[ -z $SECRETS ]]; then
+  SECRETS="$HOME/secrets"
+fi
+
 if [[ -z $WORKER_BB_SECRETS_FILE ]]; then
-  echo "Need WORKER_BB_SECRETS_FILE=[ubuntu|freebsd]_bb_secrets.json defined!"
+  echo "Need WORKER_BB_SECRETS_FILE=/path/to/[ubuntu|freebsd]_bb_secrets.json defined!"
```

Comment:
We actually need it not to be the full path if we use $SECRETS as below -- we must omit the /path/to.